src/_P118_Itho.ino

```diff
@@ -236,7 +236,7 @@ boolean Plugin_118(uint8_t function, struct EventStruct *event, String& string)
     {
       int8_t offset = -1;
 
-	  // Find matching interrupt flag
+      // Find matching interrupt flag
       for (uint8_t i = 0; i < P118_INTERUPT_HANDLER_COUNT && offset == -1; i++) {
         if (PLUGIN_118_Task[i] == event->TaskIndex) {
```

Comment:
Is this section still needed when using the datastruct setup?
Or is it needed for the interrupthandling? (thus volatile)

Maybe also have a look at the pulse counter plugin. P003.

Reply:
Ah, I tried to avoid the previous implementation limit for P003 being restricted to tasks 1..4, but that seems to be lifted by the current implementation. I can do a similar implementation, having no explicit limits, other than the number of pins supporting interrupts, just give me a little time 😃.

Reply:
Refactoring actually went smoother than first estimated 😉